doc/nrf/templates/Ecosystem_integration_template_README.rst

```diff
@@ -0,0 +1,130 @@
+.. _Ecosystem_integration:
+
+Template: Ecosystem integration
+###############################
+
+.. contents::
+   :local:
+   :depth: 2
+
+.. note::
+   Provide a short introductory description of the tool or service (third-party), indicating the value it brings to |NCS|, following these instructions:
+
+   * Give the title that starts with the tool or service (third-party) name with "integration" as a suffix, for example, "Memfault integration".
+   * Put the documentation inside the :file:`nrf/doc/nrf` folder.
+   * Add the file name in the toctree of the :file:`ext_comps` RST file.
+   * Sections with * are optional and can be left out.
+     All other sections are required.
+     Do not add new sections (unless in the sections that allow for further subsections) without discussion with the tech writer team.
+
+.. tip::
+   Provide the tool or service (third-party) in one to two sentences max (full sentences, not sentence fragments), so that the user gets a clear idea of what the tool or service (third-party) does.
+
+Partner overview*
+*****************
+
+.. note::
+   Provide an explanation of the tool or service (third-party).
+   This is optional if the tool or service (third-party) is described above.
+
+Solution architecture
+*********************
+
+.. note::
+   * Give a short description of :term:`Development Kit (DK)` and |NCS| interaction with the tool or service (third-party) and the key elements, for example, connectivity and onboarding.
+   * You can include a diagram showing how the DK and |NCS| interact with the tool or service (third-party) and the key elements.
+     This is optional.
+
+Prerequisites
+=============
+
+.. note::
+   * Provide prerequisites that must be setup before integrating with the tool or service (third-party).
+   * Mention any other tool or service (third-party) integration that needs to be setup.
+
+Before you start the |NCS| integration with AVSystem's Coiote IoT Device Management, make sure that the following are setup:
```

Comment:
```suggestion
Following is an example:
Before you start the |NCS| integration with AVSystem's Coiote IoT Device Management, make sure that the following prerequisites are completed:
```

Reply:
In the other templates, the examples are directly written without the "Following is an example:" statement.
I would keep it as is for consistency's sake.